tasks/create_fixed_routes.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Create fixed routes for all apps
+
+- include_tasks: create_fixed_route.yml
+  loop: "{{ openshift_apps }}"
```

Comment:
:thinking: I think you wanted to name `openshift_services` as `openshift_apps` in your main groupvars, right?

Reply:
You're right il a copy/paste error :$